broker-cli/scripts/install-config.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/sh
```

Comment:
[nit] unless there is a reason to use sh, most of our scripts use `#!/usr/bin/env bash`

Reply:
Ah I was just being consistent with the postinstall script in broker-cli, but I'll change.